invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/form.overrides

```diff
@@ -60,14 +56,32 @@
         position: relative;
         padding-top: 3.5rem;
 
-        .ui.icon.button {
+        .ui.icon.button.close-btn {
           position: absolute;
           top: 1rem;
           right: 1.3rem;
+          margin-top: 0;
+        }
+      }
+    }
+
+    .additional-descriptions {
+      margin-top: -1rem;
+
+      .ui.icon.button.close-btn {
+        margin-top: 0;
+      }
+
+      .ui.grid {
+        margin: 1rem 0 0 0;
+
+        .row {
+          padding-bottom: 2rem;
```

Comment:
there is a utility class you could use instead `rel-pb-2` unless it is meant to be applied to every row inside of the additional-descriptions

Reply:
Agreed to keep. Discussed in person.